packages/classic/scss/grid/_variables.scss

```
@@ -18,6 +18,7 @@ $grid-filter-cell-padding-y: $grid-filter-cell-padding-x !default;
 $grid-toolbar-padding-x: $toolbar-padding-x !default;
 $grid-toolbar-padding-y: $toolbar-padding-y !default;
 $grid-toolbar-spacing: $toolbar-spacing !default;
+$grid-toolbar-border-width: 1px !default;
```

I would suggest utilizing the `$grid-border-width` as value for the extracted var:

```suggestion
$grid-toolbar-border-width: $grid-border-width !default;
```

---

:)